scripts/android_watchdog.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+function watchdog() {
```

Comment:
Is this watchdog script added to address the case you reported when tuning session is interrupted due to crash/failure in running OpenCL kernels on Android might cause rpc server shutdown?

Reply:
Yes, we are using these watchdog scripts for rerunning rpc server on Android. `android_watchdog.sh` is used for restarting AndroidRPC application. `launch_rpc.sh` is used for restarting cpp_rpc application. `cpp_rpc` works more stable on Android in comparison with Android RPC.